src/fides/core/config/redis_settings.py

```diff
@@ -15,11 +15,14 @@ class RedisSettings(FidesSettings):
         default="utf8",
         description="Character set to use for Redis, defaults to 'utf8'. Not recommended to change.",
     )
-    db_index: Optional[int] = Field(
-        default=None,
+    db_index: int = Field(
+        default=0,
```

Comment:
I think this should be fine as a default? `None` values cause problems in the config file (because the concept of `null` doesn't exist in TOML)